sdk/core/azure-core/src/main/java/com/azure/android/core/http/interceptor/AddDateInterceptor.java

```diff
@@ -20,12 +20,13 @@
  * HTTP request.
  */
 public class AddDateInterceptor implements Interceptor {
-    private static final String DATE_HEADER = "Date";
```

Comment:
We shouldn't unhide fields just so we can access them from tests. It ends up with our tests knowing too much about our internal structure of the code under test. When writing tests, it's okay to have some duplication rather than having to DRY up everything.